### DISCUSSION TITLE
Jieba or nltk

### DISCUSSION THREAD

**Discussion by GluttonousCat** [milvus-io/milvus#38487] [Q&A and General discussion]
Does the BM25 in version2.5x support the jieba tokenizer method, i remember the method is nltk in milvus2.4x.

**Comment by yhmo** [comment]
This doc is about jieba: https://milvus.io/docs/jieba-tokenizer.md

**Comment by gangren-user** [comment]
@yhmo milvus2.5的版本，bm25检索，能否支持自定义分词文件，目前文档中虽然支持自定义分词，但是分词是通过数组的方式传递的，这种方式有一个局限性是传太多会报错，超出传输的最大限制
还有就是能否支持同义词相关搜索
另外分词器能否支持类似hanlp工具这样的分词效果，这样在某些特定的RAG检索时，可以更好的结合上下文来分词
